diff --git a/tools/rosmsg/src/rosmsg/__init__.py b/tools/rosmsg/src/rosmsg/__init__.py
index 9be35402..d1848741 100644
--- tools/rosmsg/src/rosmsg/__init__.py
+++ tools/rosmsg/src/rosmsg/__init__.py
@@ -44,7 +44,6 @@ import os
 import sys
 import yaml
 
-from catkin.find_in_workspaces import find_in_workspaces
 
 import rospkg
 import genmsg
@@ -54,6 +53,8 @@ import roslib.message
 
 from optparse import OptionParser
 
+from rules_python.python.runfiles import runfiles
+
 MODE_MSG = '.msg'
 MODE_SRV = '.srv'
 
@@ -546,11 +547,27 @@ def iterate_packages(rospack, mode):
 _catkin_workspace_to_source_spaces = {}
 _catkin_source_path_to_packages = {}
 
+def find_in_bazel_workspaces(search_dirs, pkgname, first_match_only=False):
+    runfiles_dir = os.getenv('RUNFILES_DIR')
+    if not runfiles_dir:
+        runfiles_dir = runfiles.Create().EnvVars().get('RUNFILES_DIR')
+    bear_msgs_dir = os.path.join(
+        runfiles_dir, 'bear_platform',
+        'ROS/dev_tools/pennybot_tools/src/pennybot_tools')
+    results = []
+
+    # Search in bear_msgs path
+    bear_msgs_path = os.path.join(bear_msgs_dir, 'bear_msgs')
+    if os.path.exists(bear_msgs_path):
+        results.append(bear_msgs_path)
+    return results
+
+
 def _get_package_paths(pkgname, rospack):
     paths = []
     path = rospack.get_path(pkgname)
     paths.append(path)
-    results = find_in_workspaces(search_dirs=['share'], project=pkgname, first_match_only=True, workspace_to_source_spaces=_catkin_workspace_to_source_spaces, source_path_to_packages=_catkin_source_path_to_packages)
+    results = find_in_bazel_workspaces(search_dirs=['share'], project=pkgname, first_match_only=True)
     if results and results[0].replace(os.path.sep, '/') != path.replace(os.path.sep, '/'):
         paths.append(results[0])
     return paths
